drivers/SmartThings/matter-switch/fingerprints.yml

```diff
@@ -33,6 +33,13 @@ matterManufacturer:
     productId: 0xAC
     deviceProfileName: plug-binary
 
+#Yeelight
+  - id: "Yeelight Smart Lamp"
+    deviceLabel: Yeelight Smart Lamp
+    vendorId: 0x1312
+    productId: 0x1
```

Comment:
nit: I would add a leading zero to make this `0x01` to for consistency with the other fingerprints above

Reply:
Fixed